db/migrate/20220219083748_add_users_ref_to_likes.rb

```diff
@@ -1,5 +1,5 @@
 class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
   def change
-    add_reference :likes, :author, null: false, foreign_key: foreign_key: {to_table: :users}
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] I also replaced line 3 from `add_reference :likes, :user, null: false, foreign_key: true` to `add_reference :likes, :author, foreign_key: { to_table: :users }`